app.rb

```diff
@@ -0,0 +1,121 @@
+require_relative './book'
```

Comment:
- [ ] You made great effort on this project! However your app is not running properly. It's automatically exiting when we run the app. Please fix this issue to make your project testable.


As you can see in this video:

https://user-images.githubusercontent.com/69011963/177010593-88ea1bea-833b-4e9b-8e60-ae15ecfd270f.mov